### ISSUE TITLE
启用半精度后，训练loss = nan

### ISSUE THREAD

**Issue by WuDiDaBinGe** [thu-coai/CDial-GPT#102]
Iter (loss=  nan) lr=0.0001875:   0%|                          | 5/60000 [00:09<17:37:55,  1.06s/it]08/03/2022 14:51:06
